ol7/profiles/sap.profile

```diff
@@ -0,0 +1,18 @@
+documentation_complete: true
```

Comment:
Since this is still a work in progress, documentation_complete should be ``false``.